workshop/docker/pygeoapi/docker.config.yml

```diff
@@ -233,7 +233,7 @@ resources:
 #            - type: feature
```

Comment:
Think we agreed to leave out all config sections, also the commented ones, except for the "First" slice.